dashboard/app/mailers/pd/workshop_mailer.rb

```diff
@@ -103,7 +103,7 @@ def exit_survey(workshop, teacher, enrollment)
     mail content_type: content_type,
       from: from_hadi,
       subject: 'How was your Code.org workshop?',
-      to: email_address(@teacher.name, @teacher.email)
+      to: email_address(@enrollment.name || @teacher.name, @teacher.email)
```

Comment:
Right, I think there should be a first name, last name, _and_ display name. I think first name should be split out for things like emails when we say "Hi Tanya," and then we can put it back together for certificates or matching to other things/etc. 

But I don't think we're set up for this yet? Right? So this PR can keep the existing behavior of just having the "Display name" until we can have a PR to update it across the board in all our sign up flows.